test/test-v8debugapi.js

```diff
@@ -332,7 +336,39 @@ describe('v8debugapi', function() {
         });
         process.nextTick(function() {foo(3);});
       });
+    });
+
+    it('should report error for native prop or getter', function(done) {
+      var bp = {
+        id: 'fake-id-124',
+        location: { path: 'test-v8debugapi.js', line: 8 },
+        expressions: ['process.env', 'hasGetter']
+      };
+      api.set(bp, function(err) {
+        assert.ifError(err);
+        api.wait(bp, function(err) {
+          assert.ifError(err);
+
+          var procEnv = bp.evaluatedExpressions[0];
+          assert.equal(procEnv.name, 'process.env');
+          var envVal = bp.variableTable[procEnv.varTableIndex];
+          envVal.members.forEach(function(member) {
+            assert.equal(member.varTableIndex, 1);
```

Comment:
It would be preferable to check if this is a StatusMessage with isError indicated rather than using a hard-coded index.